src/main/jni/src/genomicsdb_GenomicsDBImporter.cc

```diff
@@ -302,16 +296,10 @@ Java_org_genomicsdb_importer_GenomicsDBImporterJni_jniCopyCallsetMap
   jbyte *callsetmap_elements =
       env->GetByteArrayElements(callsetmap_as_bytearray, 0);
   int callsetmap_length = env->GetArrayLength(callsetmap_as_bytearray);
-  try {
-    callsetmap.ParseFromArray(
+  callsetmap.ParseFromArray(
         reinterpret_cast<jbyte*>(callsetmap_elements),
         callsetmap_length);
-  } catch (std::exception e) {
```

Comment:
Catching the reference to exception (const std::exception& e)  would have fixed the missing sub-class information. In this case, it doesn't matter until you're throwing the Java exception.

Reply:
Agreed. And protobuf does not throw an exception, rather returns a boolean status. We should be checking that and throwing a java exception, leave this for another pass.